Documentation/DSCResources/VMHostAdvancedSettings/VMHostAdvancedSettings.md

```diff
@@ -0,0 +1,66 @@
+# VMHostAdvancedSettings
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | Name of the Server we are trying to connect to. The Server can be a vCenter or ESXi. ||
+| **Name** | Key | string | Name of the VMHost to configure. ||
+| **Credential** | Mandatory | PSCredential | Credentials needed for connection to the specified Server. ||
+| **AdvancedSettings** | Mandatory | hashtable | Hashtable containing the advanced settings of the specified VMHost, where each key-value pair represents one advanced setting - the key is the name of the setting and the value is the desired value for the setting. ||
+
+## Description
+The resource is used to update the Advanced Settings of the specified VMHost. If the hashtable contains an invalid Advanced Setting, it is ignored. Values of type 'System.Int32' are not allowed in the hashtable and need to be cast to 'long'.
```

Comment:
Hm, can the resource convert int to long internally?

Reply:
Well if int value is not casted to long an error occurs when compiling the Configuration to MOF so the convertion cannot happen internally I am afraid.